cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -98,10 +95,14 @@ if ("${CMAKE_RANLIB}" STREQUAL "")
 	set(CMAKE_RANLIB "${EMSCRIPTEN_ROOT_PATH}/emranlib${EMCC_SUFFIX}" CACHE FILEPATH "Emscripten ranlib")
 endif()
 
-# Don't do compiler autodetection, since we are cross-compiling.
-include(CMakeForceCompiler)
-CMAKE_FORCE_C_COMPILER("${CMAKE_C_COMPILER}" Clang)
-CMAKE_FORCE_CXX_COMPILER("${CMAKE_CXX_COMPILER}" Clang)
+# CMakeForceCompiler is not to be used from CMake 3.5+
```

Comment:
Clarified this in https://github.com/kripken/emscripten/commit/777bc91e7b7c3a5291760ea8927ba5360933b779